### PR TITLE
Added Get/ListAccount endpoints

### PR DESCRIPTION
**Pull request by ieatkillerbees**

### CODE REVIEW

File: src/Actions/Account/ListAccounts.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Zend\Diactoros\Response\JsonResponse;
+use const Choredo\REQUEST_FAMILY;
+use const Choredo\REQUEST_FILTER;
+use const Choredo\REQUEST_PAGINATION;
+use const Choredo\REQUEST_SORT;
+
+class ListAccounts implements
+    EntityManagerAware,
+    Output\FractalAwareInterface,
```
Comment (greydnls) [inline src/Actions/Account/ListAccounts.php:21]: We should probably consider dropping `Interface` from our `Aware` things, so it's consistent.

File: src/Actions/Account/ListAccounts.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    {
+        return ['emailAddress', 'firstName', 'lastName'];
+    }
+
+    /**
+     * @return string[]
+     */
+    public static function getSortableFields(): array
+    {
+        return ['createdDate', 'lastLogin'];
```
Comment (greydnls) [inline src/Actions/Account/ListAccounts.php]: Feels like `firstName` and `lastName` would be valuable sort fields too, no?

File: src/Actions/Account/ListAccounts.php [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    }
+
+    public function __invoke(Request $request, Response $response, array $vars): Response
+    {
+
+
```
Comment (greydnls) [inline src/Actions/Account/ListAccounts.php]: So much whitespace, tho.

File: src/Actions/Account/ListAccounts.php [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        $filters = $request->getAttribute(REQUEST_FILTER, []);
+
+        /** @var \Choredo\Repositories\FamilyRepository $repository */
+        $repository = $this->entityManager->getRepository(Account::class);
+
+        /** @var \Doctrine\ORM\Tools\Pagination\Paginator $paginator */
+        $paginator = $repository->getAll($pagination, $sorts, $filters);
+
+        /** @var \Choredo\Entities\Family $family */
+        $family = $request->getAttribute(REQUEST_FAMILY);
```
Comment (greydnls) [inline src/Actions/Account/ListAccounts.php:68]: This is confusing me. The family is referenced in the url, but it's not used to limit accounts?

File: src/Actions/Account/ListAccounts.php (hunk 1 above)
Comment (ieatkillerbees) [reply src/Actions/Account/ListAccounts.php:21]: I agree, I thin it's better to do in a single pr though to get it all at once. I can do it in the cleanup branch in working on

File: src/Actions/Account/ListAccounts.php (hunk 2 above)
Comment (ieatkillerbees) [reply src/Actions/Account/ListAccounts.php]: :+1:

**Review by greydnls — CHANGES_REQUESTED** [review]
I think tere's a bug in the list endpoint. It should filter by family.

File: src/Actions/Account/ListAccounts.php (hunk 4 above)
Comment (ieatkillerbees) [reply src/Actions/Account/ListAccounts.php:68]: All queries for entities that have a family will automatically have a family filter applied unless explicitly disabled on the EntityManager.

**Review by greydnls — APPROVED** [review]

**Pull request merged**